src/components/books/Category.jsx

```diff
@@ -1,7 +1,18 @@
+import { useDispatch } from 'react-redux';
 import Button from '../../UI/Button';
 
 function Category() {
-  return <Button type="button" title="CHECK STATUS" />;
+  const dispatch = useDispatch();
+
+  const checkStatusHandler = () => {
+    dispatch({
+      type: 'bookstore/categories/CHECK_STATUS',
+    });
+  };
+
+  return (
+    <Button type="button" title="CHECK STATUS" onAction={checkStatusHandler} />
+  );
```

Comment:
- [ ] When testing your app I noticed that the `CHECK STATUS` button does not work, when I click on the button nothing happens. The functionality of this button is when the user clicks on it it's supposed to return the `Under construction ` text, please kindly fix that 🙃 😉

Reply:
I thought of that, but there is no component to display that